content/applications/hr/frontdesk.rst

```diff
@@ -0,0 +1,318 @@
+=========
+Frontdesk
+=========
+
+The Odoo *Frontdesk* application is a way for visitors to check in to a building or location and
+alert the person they are seeing that they have arrived. Additionally, they can request a
+pre-configured beverage, to be brought to them while they wait. This application is ideal for
+businesses that do not have someone working at a reception desk, or perhaps do not have a lobby or
+waiting area available to guests and visitors.
+
+Configuration
+=============
+
+The first item to be configured to use the *Frontdesk* application is the station or stations,
+followed by the drinks that are offered.
```

Comment:
What do you think about moving this sentence inside the heading below, since it's relevant to Stations? Totally optional, and up to you! For example, it would look like:

Configuration
=========

Stations
---------

The first item to be configured ...

Reply:
I was told it was best to try and have some text between sections, always- so I think I'll leave this here since there are just 2 configurations and it explains that you do them both.